src/software/ai/play_selection_fsm.h

```diff
@@ -110,14 +108,32 @@ struct PlaySelectionFSM
 
             // Check for transitions to other states, if not then default to running the
             // current play
-            SetPlay_S + Update_E[gameStateHalted_G] / setupHaltPlay_A     = Halt_S,
-            SetPlay_S + Update_E[gameStateStopped_G] / setupStopPlay_A    = Stop_S,
-            SetPlay_S + Update_E[gameStatePlaying_G] / setupOffensePlay_A = Playing_S,
+            SetPlay_S + Update_E[gameStateHalted_G] / (resetSetPlay_A, setupHaltPlay_A) =
+                Halt_S,
```

Comment:
Didn't know this was possible with boost SML. does this call both actions?

Reply:
Yep, and I think in that order as well